src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )
+      }
+
+      <p className="Person__partner">
+        {!isMarried
+          ? 'I am not married'
+          : `${partnerName} is my ${sex === 'm'
+            ? 'wife'
+            : 'husband'}`
+        }
+      </p>
```

Comment:
Try to avoid nested ternary, it is better to use variable to determine `wife` or `husband`, also create constant for message if person is not married. For example `const NOT_MARRIED_MESSAGE = '...'`